lib/compat/wordpress-6.1/block-patterns/header-with-large-font-size.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Header: Header with Large Font Size.
+ *
+ * @package WordPress
+ */
+
+return array(
+	'title'      => _x( 'Header with Large Font Size', 'Block pattern title' ),
+	'blockTypes' => array( 'core/template-part/header' ),
+	'categories' => array( 'header' ),
+	'content'    => '<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"30px","right":"30px","bottom":"30px","left":"30px"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
+					<div class="wp-block-group alignfull" style="padding-top:30px;padding-right:30px;padding-bottom:30px;padding-left:30px;"><!-- wp:site-title {"style":{"typography":{"fontStyle":"normal","fontWeight":"700"}},"fontSize":"large"} /-->
+					
+					<!-- wp:navigation {"ref":31,"layout":{"type":"flex","justifyContent":"space-between"},"style":{"spacing":{"blockGap":"30px"}},"fontSize":"large"} /--></div>
```

Comment:
I believe the practice is to remove the ref -There might not be a navigation menu with the ID "31" on the install.

Reply:
Yes, these aren't meant to be here (see "Issues" in the PR description).